### PR TITLE
update the sw.yml

### PR DESCRIPTION
**Pull request by Anshulverma-prg**
change the name and change the os

### CODE REVIEW

File: .github/workflows/sw.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: sw
+name: as
```
Comment (stweil) [inline .github/workflows/sw.yml:1]: What is the reason for this change?

**Comment by Anshulverma-prg** [comment]
Because it is bulid the new program name and it will be necessary for the
program.

**Comment by zdenop** [comment]
@egorpugin : any idea about this?

**Comment by egorpugin** [comment]
Maybe @Anshulverma-prg tried to do changes only in his fork?
1. Name is fine.
2. We'll use Ubuntu LTS, no need to update.
